### PR TITLE
Mise à jour de l’adresse postale, fix #2

### LINKED ISSUE #2: Mettre à jour l'adresse postale
Bonjour,

Il me semble que l'adresse postale est mainteannt 14 avenue ledru-rollin 75012 Paris. Est-ce que mon information n'est pas bonne ?

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -19,9 +19,8 @@ <h1><a href="/">Association Agile France</a></h1>
 		<footer>
 			<p>
```
Comment (MattiSG) [inline _layouts/default.html]: L'adresse postale ainsi que l'adresse email devraient être englobées dans un [élément `<address>`](https://www.w3.org/TR/2014/PR-html5-20140916/sections.html#the-address-element) au lieu du `<p>`. Cet élément devrait également avoir un attribut [`translate="no"`](https://www.w3.org/TR/2014/PR-html5-20140916/dom.html#the-translate-attribute) pour en empêcher la traduction automatique.

File: _layouts/default.html [hunk 2]
```diff
@@ -19,9 +19,8 @@ <h1><a href="/">Association Agile France</a></h1>
...
 				Association Agile France<br>
-				chez Numa<br>
-				39 rue du Caire<br>
-				75002 Paris<br>
```
Comment (MattiSG) [inline _layouts/default.html]: Il me semblerait utile d'ajouter `France` au vu de la vocation internationale d'AgileFrance 🙂

**Review by MattiSG — CHANGES_REQUESTED** [review]

File: _layouts/default.html (hunk 1 above)
Comment (davidbgk) [reply _layouts/default.html]: Done with the email but I disagree with the postal address:

> The address element must not be used to represent arbitrary addresses (e.g. postal addresses), unless those addresses are in fact the relevant contact information. (The p element is the appropriate element for marking up postal addresses in general.)

Comment (MattiSG) [reply _layouts/default.html]: > unless those addresses are in fact the relevant contact information

:wink:

Aren't they? I think both the postal and email addresses are the relevant contact information for the “nearest `body` ancestor”. See the [example on MDN](https://developer.mozilla.org/fr/docs/Web/HTML/Element/address).

I read the spec's emphasis on “no arbitrary address” as in “this is not a generic postal address tag”, not as in “no postal address”.

Comment (davidbgk) [reply _layouts/default.html]: Fair enough.

**Review by MattiSG — DISMISSED** [review]
🎉

**Comment by davidbgk** [comment]
No merge?

**Comment by MattiSG** [comment]
Was closed accidentally due to target branch deletion. Reopening.

**Review by MattiSG — APPROVED** [review]

**Pull request merged**